core/src/test/scripts/test_data/study_es_0/result_report.html

```diff
@@ -1259,6 +1259,13 @@ <h4 class="panel-title">data_mutations_extended.maf</h4>
                 <td>Line will not be loaded due to the variant classification filter. Filtered types: [Silent, Intron, 3&#39;UTR, 3&#39;Flank, 5&#39;UTR, 5&#39;Flank, IGR, RNA]</td>
                 <td>Silent</td>
               </tr>
+              <tr class="warning">
+                <td><span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span><span class="sr-only">Warning</span></td>
+                <td>10, 11, 16, (9 more)</td>
+                <td>&ndash;</td>
+                <td>Off panel variant. Gene symbol not known to the targeted panel.</td>
+                <td>MSH3, MYB, TP53, (4 more)</td>
+              </tr>
```

Comment:
Same as the last comment, I don't think we expect to see warnings when validate `study_es_0`. I don't really understand why this warning shows up since I saw those genes in the `gene-panels.json`.

Reply:
Thanks for pointing this. I wasn't sure if the test study should succeed 100% or can succeed with warnings. Added missing genes to the test gene panel and the study passed with no warnings. Revered the reports.